src/app/zap-templates/zcl/data-model/chip/thread-network-directory-cluster.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+    <domain name="CHIP"/>
+
+    <struct name="ThreadNetworkStruct">
```

Comment:
This does not match the type name in the spec, but presumably the spec is planned to be fixed here?

Reply:
Fixing in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9279